.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
```

Comment:
Do we want to put this setup as a tool in `pyproject.toml` instead?

Reply:
Since you're the second person to ask, I've moved it over to `pyproject.toml` 😄